app/equipment/page.tsx

```
@@ -0,0 +1,31 @@
+import BorrowingEquipList from "@/components/borrowing-equip-list";
+import EquipmentCell from "@/components/equipment";
+import { getEquipmentById } from "@/db/queries/equipments";
+import styles from "./base.module.css";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)